subsystems/networking/setup.go

```diff
@@ -151,3 +152,17 @@ func (n *Networking) enableWifi(ctx context.Context) error {
 		}
 	}
 }
+
+func (n *Networking) initBluetooth() error {
+	if n.bluetoothService != nil {
+		return nil
+	}
+	deviceName := fmt.Sprintf("%s-%s-%s", n.Config().Manufacturer, n.Config().Model, n.Config().FragmentID)
```

Comment:
Are there constraints on the device name? Manufacturer and Model could both be rather long, and fragment ID is a full 32-byte UUID, so that'll look really weird on a phone. Also, all robots with the same config would look the same, so you couldn't tell them apart. I would suggest using the same pattern as the wifi. That is, prefix+hostname: 

`n.Config().HotspotPrefix + "-" + strings.ToLower(hostname)`
You can get the hostname from `os.Hostname()`